common-service-core/src/main/java/com/ca/mfaas/message/api/ApiMessageView.java

```diff
@@ -7,31 +7,37 @@
  *
  * Copyright Contributors to the Zowe Project.
  */
-package com.ca.mfaas.rest.response;
+package com.ca.mfaas.message.api;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Data;
 
 import java.util.List;
 
 /**
- * This interface is intended for REST API responses that contain error,
+ * This class is intended for REST API responses that contain error,
  * warning, or informational messages in the common MFaaS format.
  *
  * It is preferred to return successful responses without messages if possible
  * and use only plain responses without wrapping for them.
  *
- * The {@link ApiMessage} and its implementation {@link com.ca.mfaas.rest.response.impl.BasicApiMessage} should be used
+ * Its implementation {@link ApiMessage} should be used
  * in the case when a problem (an error) happens and then the response contains only the error(s).
  *
  * When a response needs to contain both data and messages (e.g. warnings)
  * then it is adviced for the response class to implement {@link ApiMessage} too.
```

Comment:
"When a response needs to contain both data and messages (e.g. warnings), implement {@link ApiMessage}," Makes sense?

Reply:
I think we can delete all this part, since it doesn't bring any useful information and it's a bit confusing.
```
* When a response needs to contain both data and messages (e.g. warnings)
 * then it is adviced for the response class to implement {@link ApiMessage} too.
 * But this should be an exception and we should try to make the REST API easy to use without
 * the need for the API user to process informational and warning messages.`
```

Reply:
If you say it brings no value, I don't mind scrapping it.